schemas/dev/subject.schema.json

```diff
@@ -70,6 +95,27 @@
       "maximum": 120,
       "pattern": "[<]\\d|[>]\\d|\\d+[-]\\d+"
     },
+    "date_of_birth": {
+      "type": "string",
+      "format": "date",
+      "description": "Date of birth",
+      "category": "demographics"
+    },
+    "dob_year": {
+      "type": "number",
```

Comment:
-> integer for dob parts, also recommend using min/max for validation for month and day, and maybe a minimum of 1900 for the year (max age permitted is ~120 years)